helm/prometheus-meta-operator/templates/configmap.yaml

```diff
@@ -24,9 +24,12 @@ data:
       prometheus:
         address: {{ .Values.Installation.V1.Monitoring.Prometheus.Address }}
         baseDomain: {{ .Values.Installation.V1.Monitoring.Prometheus.Host }}
+        retention:
+          duration: 2w
+          size: 90GB
         storage:
           createPVC: {{ and .Values.Installation.V1.Monitoring.Prometheus.Storage.Enabled .Values.Installation.V1.Monitoring.Prometheus.Storage.CreatePVC }}
-          size: 20Gi
```

Comment:
Resizing pvc via statefulset is not supported yet in kubernetes: https://github.com/kubernetes/kubernetes/issues/68737

Reply:
I am not sure what you want me to change here. We still need to increase it to 100Gi

Reply:
I'm just saying this does not work for existing cluster. The volume size won't be changed. So we need a workaround.

Reply:
So we would need a volume deleter if the volume size is 20Gi right?

Reply:
Yeah right, official doc says you have to manually delete the pvc created by a stateful set when cleaning it (also valable for resizing): https://kubernetes.io/docs/tasks/run-application/delete-stateful-set/#complete-deletion-of-a-statefulset